### PR TITLE
attempt to build against the new nexus

### PR DESCRIPTION
**Pull request by metas-ts**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env groovy
 
 // note that we set a default version for this library in jenkins, so we don't have to specify it here
-@Library('misc')
+@Library('misc@intensive_care_uat_newnexus')
```
Comment (metas-ts) [inline Jenkinsfile:4]: revert the changes in jenkinsfile before integrating

File: misc/services/camel/de-metas-camel-edi/buildfile.groovy [hunk 2]
```diff
@@ -51,12 +51,8 @@ def build(final MvnConf mvnConf, final Map scmVars, final boolean forceBuild = f
             'de-metas-edi-esb-camel', // artifactName
             env.BRANCH_NAME, // branchName
             env.MF_VERSION, // versionSuffix
-            './', // workDir
-            '', // additionalBuildArgs
-            'Dockerfile', // dockerFile
-            '', // pullRegistry - empty string means "use dockerhub!"
-            'dockerhub_metasfresh' // pullRegistryCredentialsId
-    ) 
+            '.' // workDir
```
Comment (metas-ts) [inline misc/services/camel/de-metas-camel-edi/buildfile.groovy:54]: keep when integrating

**Comment by teosarca** [comment]
@metas-ts  is this active?

**Comment by teosarca** [comment]
closing, legacy
